src/Logging/Logging.File/Extensions/LogServiceFactory.File.Extensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using CodeMonkeys.Logging.File;
+
+namespace CodeMonkeys.Logging
+{
+    public static partial class LogServiceFactoryExtensions
+    {
+        /// <summary>
+        /// Method for adding the <see cref="ServiceProvider"/> to the <see cref="LogService"/>.
+        /// </summary>
+        public static void AddFile(this ILogServiceFactory _this)
+        {
+            var provider = new ServiceProvider();
+            _this.AddProvider(provider);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter '_this' before using it.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5626772)